.github/actions/templates/validateModuleDeployment/action.yml

```diff
@@ -1,3 +1,31 @@
+#########################################################
+## 'Deploy module' Composite Action                    ##
+#########################################################
+##
+## This composite action enables you to both test and execute a deployment of a given template and parameter file.
+## It can further optionally remove/not remove the deployment
```

Comment:
What is the default? should it be remove by default?

Reply:
As per the parameters table below: `true`

Reply:
Is this what we want? If I'm not mistaken we have true for ADO

Reply:
nvm, we have true here and false for ADO. We'll need to update that for consistency, but out of this pr